hack/annotate.sh

```diff
@@ -31,6 +31,13 @@ function annotate() {
 			fi
 		fi
 
+		for attempt in $( seq 1 2 ); do
+			if oc annotate -n "${namespace}" "is/${name}" "release.openshift.io/config=$( cat "${conf}" | jq -c . )" --overwrite; then
+				return
+			fi
+			sleep 5
+		done
+		# Last attempt must be successful
```

Comment:
Why? Can we just use sth like this https://github.com/kubermatic/kubermatic/blob/00c0da788d618a4fbf3ddf1e9655c8a3a06d0a28/hack/lib.sh#L28-L38 ?

Reply:
Well last attempt must be successful b/c when it runs it implies the previous ones weren't (if they would, we would return and never got here). I can as well copy that piece of bash from there, but we don't get too much different behavior (just the backoff) /shrug

Reply:
Done